.travis.yml

```diff
@@ -65,7 +68,7 @@ before_script:
             echo "Not attempting to run doctr due to the docs failed to build"
         else
             pip install doctr
-            doctr deploy --built-docs docs/build/html .
+            doctr deploy --built-docs docs/build/html . --branch-whitelist main
```

Comment:
After looking at the corresponding code of [`doctr`](https://github.com/drdoctr/doctr/blob/1e5be24700ca5b07fa1603ccc4b89977dbd1cd78/doctr/__main__.py#L306), this looks like the right solution 👍